src/Components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => (
+  <>
+    <div className="footer">
+      <div>
+        <p>&copy; 2022, E-commerce Website for your online sales.</p>
+        <p>
+          Contact info:
+        </p>
+        <div className="info">
+          <span>Tel: +393298592707</span>
+          <span>isigba.ambrose94@gmail.com</span>
+          <span>https://www.linkedin.com/in/ambrose-isigba/</span>
+          <span>Ambrosegithub</span>
+          <span>twitter.com/home</span>
+          <span> angel.co/u/ambrose-isigba</span>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] I have noticed that your footer has a fixed position and as a result, it is sticking to the windows screen and lying behind other elements and this is not appropriate. 
![image](https://user-images.githubusercontent.com/90222110/189452981-ea24b07d-6de9-44e5-ad30-47b95f6a456b.png)

As a convention, a footer is supposed to stay at the bottom of the page and this can be achieved with the absolute positioning accompanied by other styling rules.  You can check out [this article](https://stackoverflow.com/questions/643879/css-to-make-html-page-footer-stay-at-bottom-of-the-page-with-a-minimum-height-b) and [this one](https://dev.to/nehalahmadkhan/how-to-make-footer-stick-to-bottom-of-web-page-3i14)  😎 🏃‍♂️